src/v2/Components/ArtworkFilter/Utils/getSelectedFIltersCounts.ts

```diff
@@ -0,0 +1,72 @@
+import { isArray, isEmpty } from "lodash"
+import {
+  ArtworkFilters,
+  FilterParamName,
+  initialArtworkFilterState,
+  SelectedFiltersCounts,
+} from "../ArtworkFilterContext"
+
+const waysToBuyFilterNames = [
+  FilterParamName.waysToBuyBuy,
+  FilterParamName.waysToBuyMakeOffer,
+  FilterParamName.waysToBuyBid,
+  FilterParamName.waysToBuyInquire,
+]
+
+const rangeFilterNames = [
+  FilterParamName.width,
+  FilterParamName.height,
+  FilterParamName.priceRange,
+]
+
+export const getSelectedFiltersCounts = (selectedFilters: ArtworkFilters) => {
+  const counts: Partial<SelectedFiltersCounts> = {}
+  const filtersParams = Object.values(FilterParamName)
+
+  Object.entries(selectedFilters).forEach(([paramName, paramValue]) => {
+    if (!filtersParams.includes(paramName as any)) {
+      return
+    }
+
+    switch (true) {
+      case waysToBuyFilterNames.includes(paramName as FilterParamName): {
+        if (paramValue) {
+          counts.waysToBuy = (counts.waysToBuy ?? 0) + 1
+        }
+        break
+      }
+      case rangeFilterNames.includes(paramName as FilterParamName): {
+        if (paramValue !== "*-*") {
+          counts[paramName] = 1
+        }
+        break
+      }
+      case paramName === FilterParamName.artistsIFollow: {
+        counts.artistIDs = (counts.artistIDs ?? 0) + 1
+        break
+      }
+      case isArray(paramValue): {
+        if (isEmpty(paramValue)) {
+          break
+        }
+
+        const isArtistsFilter = paramName === FilterParamName.artistIDs
+        const countToAdd = isArtistsFilter ? counts.artistIDs ?? 0 : 0
```

Comment:
**question**: can't we just use `const countToAdd = counts.artistIDs ?? 0` here ?

Reply:
We check if it is artists filter because we have filter `artistsIFollow` that is processed separately. So if we have `artistsIFollow: true` we should add all artist filter options to `counts.artistsID`.

I think we can replace it with:
```
if (paramName === FilterParamName.artistIDs) {
  counts.artistIDs = paramValue.length + counts.artistIDs ?? 0
  break
}

counts[paramName] = paramValue.length
break
```